### ISSUE TITLE
Orphaned Form Label

### ISSUE THREAD

**Issue by erinesullivan** [erinesullivan/gbooks-challenge#1]
# WAVE Overview
## What It Means
A form label is present, but it is not correctly associated with a form control.

## Why It Matters
An incorrectly associated label does not provide functionality or information about the form control to the user. It usually indicates a coding or other form labeling issues.

## How to Fix It
Properly associate the label with its corresponding form control. If there is no corresponding form control, remove the label. Labels are not appropriate for image, submit, reset, button, or hidden form controls.

## The Algorithm... in English
A `<label>` element:
* does not surround a form control and the for attribute is missing/empty
* references an element that is not present in the page
* references an element that is not an `<input>`, `<select>` or `<textarea>` element
* references an `<input>` element with image, submit, reset, button, or hidden type

## Standards and Guidelines
* [Section 508 (n)](http://webaim.org/standards/508/checklist#standardn)
* [1.1.1 Non-text Content (Level A)](http://webaim.org/standards/wcag/checklist#sc1.1.1)
* [1.3.1 Info and Relationships (Level A)](http://webaim.org/standards/wcag/checklist#sc1.3.1)
* [2.4.6 Headings and Labels (Level AA)](http://webaim.org/standards/wcag/checklist#sc2.4.6)
* [3.3.2 Labels or Instructions (Level A)](http://webaim.org/standards/wcag/checklist#sc3.3.2)

**Comment by erinesullivan** [comment]
[Resolved](https://github.com/erinesullivan/gbooks-challenge/pull/3).

**Comment by erinesullivan** [comment]
[Resolved](https://github.com/erinesullivan/gbooks-challenge/pull/3).
